subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -607,7 +607,7 @@ uint8_t ll_adv_params_set(uint16_t interval, uint8_t adv_type,
 				sizeof(struct pdu_adv_adi));
 
 			adi = (void *)pri_dptr;
-			adi->sid = sid;
+			adi->did_sid_packed[1] = sid + (adi->did_sid_packed[1] & 0xF0);
```

Comment:
I think it would be nicer to use a macro `PDU_ADV_SID_SET` here.

Reply:
Added macros covering this